src/app/resources/browser/styles/manual.css

```
@@ -0,0 +1,193 @@
+  }
+  .doc section {
+    padding-top: 1.25rem;
+    scroll-margin-top: 0;
+  }
+  .doc .table-wrap {
+    overflow: visible;
+  }
+  .doc h2,
+  .doc h3 {
+    break-after: avoid;
+    page-break-after: avoid;
```

Stylelint (17.14.0)

[error] Expected "page-break-after" to be "break-after" (property-no-deprecated)

(property-no-deprecated)